addons/binding/org.openhab.binding.kodi/src/main/java/org/openhab/binding/kodi/internal/protocol/KodiConnection.java

```diff
@@ -365,6 +369,12 @@ private void processPlayerStateChanged(String method, JsonObject json) {
             if (data.has("item")) {
                 JsonObject item = data.get("item").getAsJsonObject();
                 String mediaType = item.get("type").getAsString();
+                if (mediaType.equals("channel") && item.has("channeltype")) {
+                    String channelType = item.get("channeltype").getAsString();
+                    if (channelType.equals("radio")) {
+                        mediaType = "radio";
+                    }
+                }
                 listener.updateMediaType(mediaType);
```

Comment:
@pail23 @MHerbst I have a question to this line of code (and the following). Why is the "mediatype" channel updated here? The following method "requestPlayerUpdate" will update it too, if the second parameter is set to "true". I like to refactor this part and remove updating the "mediatype" channel here.